workers/unity/Packages/io.improbable.gdk.tools/GenerateCode.cs

```diff
@@ -245,6 +230,36 @@ private static void Generate()
             }
         }
 
+        private static string GetDefaultLoggerOutputPath(GdkToolsConfiguration toolsConfig = null)
+        {
+            toolsConfig = toolsConfig ?? GdkToolsConfiguration.GetOrCreateInstance();
+            return Path.GetFullPath(Path.Combine(toolsConfig.CodegenLogOutputDir, "codegen-output.log"));
+        }
```

Comment:
Should this be a method on the `GdkToolsConfiguration` class?

Reply:
moved this + getting the codegen output path to properties on tools config class